chronos/asyncfutures2.nim

```diff
@@ -60,6 +61,11 @@ type
       closure*: iterator(f: Future[T]): FutureBase {.raises: [Defect, CatchableError, Exception], gcsafe.}
     value: T ## Stored value
 
+  ## Future with a tuple of possible exception types
+  ## eg RaiseTrackingFuture[void, (ValueError, OSError)]
+  ## Should generally not be used manually
```

Comment:
here, we should document that one should annotate with `asyncraises` instead of using "manually", for callback type definitions for example - also, afair shouldn't type comments come below the type, indented?

Reply:
Done